vendor/github.com/hashicorp/nomad/api/tasks.go

```diff
@@ -632,6 +632,7 @@ type DispatchPayloadConfig struct {
 
 const (
 	TaskLifecycleHookPrestart = "prestart"
+	TaskLifecycleHookPoststart = "poststart"
```

Comment:
Curious if this was meant to be included in this PR?

Reply:
Yes - this is orthogonal but needed because https://github.com/hashicorp/nomad/pull/8390/files updated api/ package without running `make sync`.